src/stronghold.hs

```diff
@@ -237,13 +237,14 @@ site runStoreOp =
       hier <- Trees.lastHierarchy hist
       snd <$> Trees.materializedView path hier
 
-    let etag = (Base16.encode . hash . toStrict . Aeson.encode) json
+    let jsonBS = (toStrict . Aeson.encode) json
+    let etag = (Base16.encode . hash) jsonBS
```

Comment:
I liked my little transformation piplines. Fixed in 57c0eae0379da0228cbb839a234a99f1b0ed5392